tools/releaseBuild/azureDevOps/templates/linux.yml

```diff
@@ -94,6 +131,24 @@ jobs:
         PackageVersion: $(Version)
         sourceScanPath: '$(PowerShellRoot)/tools'
 
+  - ${{ if eq(variables.build,'deb') }} :
+    - template: EsrpSign.yml@ComplianceRepo
+      parameters:
+        buildOutputPath: '$(System.ArtifactsDirectory)/pwshLinuxBuildMinSize'
+        signOutputPath: '$(System.ArtifactsDirectory)/pwshLinuxBuildMinSize'
+        certificateId: "CP-230012"
+        pattern: |
+          **\*.dll
+          **\*.psd1
+          **\*.psm1
+          **\*.ps1xml
+          **\*.ps1
+          **\*.exe
```

Comment:
There is no `*.exe` files in Linux build.

Reply:
```suggestion
```
ELF files cannot be authenticode signed so there is nothing to replace this with.